uwblueprint/community-fridge-kw
# feat: email cron job

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Email-Reminders-Cron-Job-9d8cdba9a68141728d5696759d3f7c06)



## Implementation description
* Added a cron job which sends emails



## Steps to test
1. Adjust values in implementation/emailService.ts line 29 to '* * * * *'
2. Add values to the scheduling table
3. Check if emails were sent



## What should reviewers focus on?
* 


## Checklist
